### PR TITLE
Fixed rotated computational grid.

### PR DESCRIPTION
**Pull request by cjvogl**
Found an error where the cells were shifted horizontally and not lined up with the fault.  Also allowed for rectangular cells so that both the fault width and fault depth divide evenly by the corresponding horizontal/vertical cell spacing.

### CODE REVIEW

File: 2d/sloping_fault/setprob.f90 [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+!     # open the unit with new routine from Clawpack 4.4 to skip over
+!     # comment lines starting with #:
+      call opendatafile(iunit, fname)
+
+
+!
+      read(7,*) width ! domain_depth
+      read(7,*) width ! domain width
+      read(7,*) center(1)
+      read(7,*) width
```
Comment (rjleveque) [inline 2d/sloping_fault/setprob.f90:27]: Why is width read three times and appears to have the value of fault_depth at the end?

File: 2d/sloping_fault/setaux.f90 [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+          areap = areap - (ypcorn(3)-ypcorn(1))*(xpcorn(2)-xpcorn(4))
+          areap = 0.5*dabs(areap)
+          aux(12,i,j) = areap/(dx*dy)
+
+          ! set slip:
+          xccorn(1) = xlower + float(i-1)*dx
+          yccorn(1) = ylower + float(j-1)*dy
+          if ((abs(yccorn(1)-center(2)) < 0.5d0*dy) .and. &
+              (xccorn(1) >= xcb(1)) .and. (xccorn(1) <= xcb(2))) then
+                aux(13,i,j) = exp(-((xccorn(1)-center(1))/(center(1)-xcb(1))**2)
```
Comment (rjleveque) [inline 2d/sloping_fault/setaux.f90]: This line is missing a ) at the end.

File: 2d/sloping_fault/setprob.f90 (hunk 1 above)
Comment (cjvogl) [inline 2d/sloping_fault/setprob.f90:27]: It's just a way to read past the domain_depth and domain_width lines in setprob.data.  There is probably a more slick/readable way to do this (by maybe reading those into a "temp" variable that is never used again).  Let me know if you think there's a better way.

File: 2d/sloping_fault/setaux.f90 (hunk 2 above)
Comment (cjvogl) [inline 2d/sloping_fault/setaux.f90]: Corrected in subsequent commit.  Thanks for catching that.

**Pull request closed without merging**